randoc.js

```diff
@@ -2,7 +2,16 @@ const chance = new require('chance')();
 
 const specialRandomDocument = (fieldName, schema) => {
   if (schema._exists && !chance.bool({ likelihood: schema._exists }) ) {
-    return {};
+    return {
+      ...(schema._arrayOf && { [fieldName]: [] })
+    };
+  }
+  if (schema._arrayOf !== undefined && chance[schema._type]) {
```

Comment:
Elsewhere this library [defaults to `string`](https://github.com/rouanw/randoc/blob/d9c3cdb9c9846f3bd5a2b7cc9a98bc64ddb4b58c/randoc.js#L28-L29) if the type doesn't end up being a real chance.js function. Maybe we could extract that logic and use it wherever we call chance with a type that may or may not exist?

Reply:
Sounds good. I made that change as well. But for the array I'm still returning an array with one Change string in it, so it's the same type.